util/dvsim/SimCfg.py

```diff
@@ -551,6 +554,22 @@ def cov_analyze(self):
         for item in self.cfgs:
             item._cov_analyze()
 
+    def _cov_unr(self):
+        '''Use the last regression coverage data to generate unreachable
+        coverage exclusions.
+        '''
+        # Create initial set of directories, such as dispatched, passed etc.
```

Comment:
Add a check here to ensure tool == vcs. It should fail immediately and cleanly if I pass `--cov-unr -t xcelium`.

Reply:
Added, thanks